docs/tech-reference/distributor.rst

```diff
@@ -160,3 +160,77 @@ Example Redirect File Contents::
     ],
   "tags": {"latest": "769b9341d937a3dba9e460f664b4f183a6cecdd62b337220a28b3deb50ee0a02"}
  }
+
+Docker rsync Distributor
+------------------------
+
+Purpose:
+--------
+The Docker rsync distributor publishes docker content to a remote server. The distributor uses
+rsync over ssh to perform the file transfer. Docker images (v1) are published into the root of
+the remote repository. Manifests (v2) are published into ``manifests`` directory and Blobs (v2) are
+published into ``blobs`` directory. The docker rsync distributor is supposed to be run prior to
+publishing with the docker web distributor.
+
+Configuration
+=============
+Here is an example docker_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "my_docker_rsync_distributor",
```

Comment:
I think this example is missing a `postdistributor_id`. I *think* that field is a required field.